src/api.ts

```diff
@@ -0,0 +1,43 @@
+import sampleActivities from "./data";
+
+export type ActivityDefinition = any;
+
+export const getActivityDefinition = (activity: string, baseUrl?: string | null): Promise<ActivityDefinition> => {
+  return new Promise((resolve, reject) => {
+    if (!baseUrl) {
+      if (sampleActivities[activity]) {
+        setTimeout(() => resolve(sampleActivities[activity]), 250);
+      } else {
+        reject(`No sample activity matches ${activity}. For an authored activity, use "baseUrl=..."`);
+      }
+    } else {
+      const urlRegex = /^(https:|http:)\/\/\S+/;
+      if (baseUrl.match(urlRegex)) {
+        getActivityDefinitionFromLara(activity, baseUrl).then(resolve);
+      } else {
+        reject(`${baseUrl} must be a valid url, e.g. baseUrl=https%3A%2F%2Fauthoring.concord.org`);
+      }
+    }
+  });
+};
+
+// http://app.lara.docker/activities/5/pages/10/export
```

Comment:
Is this an example URL? A little context would be useful.

Reply:
...nope, that's a comment to myself while I was working... Begone!